crates/ruff_db/Cargo.toml

```diff
@@ -13,6 +13,7 @@ license = { workspace = true }
 [dependencies]
 ruff_annotate_snippets = { workspace = true }
 ruff_cache = { workspace = true, optional = true }
+ruff_diagnostics = { workspace = true }
```

Comment:
I think we should move `Fix` and `Edit` (and remove `ruff_diagnostics`?). But it was the right move to not do this as part of this PR